pkg/transformers/config/defaultconfig/varreference.go

```diff
@@ -105,5 +105,11 @@ varReference:
 
 - path: spec/tls/hosts
   kind: Ingress
+
+- path: metadata/labels
+  kind: 
```

Comment:
Well, that's interesting :)

Reply:
:) removed, it was the product of testing the `Gvk.IsSelected` and `yaml` parser
I removed it